components/brave_rewards/resources/BUILD.gn

```diff
@@ -58,16 +58,22 @@ repack("resources") {
   public_deps = [
     ":page_generated_resources",
     ":internals_generated_resources",
-    ":tip_generated_resources",
     ":static_resources",
   ]
 
+  if (!is_android) {
+    public_deps += [ ":tip_generated_resources" ]
+  }
+
   sources = [
     "$root_gen_dir/brave/components/brave_rewards/resources/brave_rewards_page_generated.pak",
     "$root_gen_dir/brave/components/brave_rewards/resources/brave_rewards_internals_generated.pak",
-    "$root_gen_dir/brave/components/brave_rewards/resources/brave_rewards_tip_generated.pak",
     "$root_gen_dir/brave/components/brave_rewards/resources/brave_rewards_static.pak",
   ]
 
+  if (!is_android) {
+    sources += [ "$root_gen_dir/brave/components/brave_rewards/resources/brave_rewards_tip_generated.pak" ]
```

Comment:
Please group these together in one !is_android block. It's better to group the sources and deps together anyway